### DISCUSSION TITLE
Anyone having problems with Python CDK connectors following an update to 0.50.33?

### DISCUSSION THREAD

**Discussion by enricojr01** [airbytehq/airbyte#32585] [Connector Questions]
So we've just recently updated to 0.50.33 at work, and one of our custom connectors finishes sync without issue, but the output is completely empty. 

I'm currently testing this locally, and when output to /tmp via the Local JSON destination the output looks like this:

```
{"_airbyte_ab_id":"3b0a920b-42bd-4459-b530-91dd5a5c0181","_airbyte_emitted_at":1699949786011,"_airbyte_data":{}}
{"_airbyte_ab_id":"9cc946f7-c51c-4496-8167-6abea2fc67ec","_airbyte_emitted_at":1699949786012,"_airbyte_data":{}}
{"_airbyte_ab_id":"70184c73-6429-428f-a100-04e280c96090","_airbyte_emitted_at":1699949786012,"_airbyte_data":{}}
{"_airbyte_ab_id":"52788d0e-2e73-4aba-bf76-b08ce8d4a5e7","_airbyte_emitted_at":1699949786013,"_airbyte_data":{}}
{"_airbyte_ab_id":"8d060937-0e50-4266-b317-64503c8644d5","_airbyte_emitted_at":1699949786014,"_airbyte_data":{}}
{"_airbyte_ab_id":"56ad2e38-c78e-48df-b6b4-eaa2b1f1edac","_airbyte_emitted_at":1699949786015,"_airbyte_data":{}}
{"_airbyte_ab_id":"af8cc04d-b7f4-46f0-905d-c445f58ddcba","_airbyte_emitted_at":1699949786016,"_airbyte_data":{}}
{"_airbyte_ab_id":"bfa2bc3d-9898-4acc-9c6d-f6c1ce361a1d","_airbyte_emitted_at":1699949786016,"_airbyte_data":{}}
{"_airbyte_ab_id":"c9cdfdd7-a60a-4d13-884e-418e55b6613d","_airbyte_emitted_at":1699949786017,"_airbyte_data":{}}
{"_airbyte_ab_id":"36ad408d-c3f3-48c9-b548-93444aa7a656","_airbyte_emitted_at":1699949786018,"_airbyte_data":{}}
{"_airbyte_ab_id":"b4cadda9-0125-4779-a075-eae03edc3e29","_airbyte_emitted_at":1699949786019,"_airbyte_data":{}}
{"_airbyte_ab_id":"4eea18c0-9cfc-4963-8498-3de7620a5f22","_airbyte_emitted_at":1699949786020,"_airbyte_data":{}}
{"_airbyte_ab_id":"0f1d6d7e-a09f-47ad-906b-b03e244c9424","_airbyte_emitted_at":1699949786021,"_airbyte_data":{}}
{"_airbyte_ab_id":"fa6e9e8a-4f12-46f8-b04c-58e435ff02b7","_airbyte_emitted_at":1699949786021,"_airbyte_data":{}}
{"_airbyte_ab_id":"d2ef6f51-9e9e-40a0-85a4-a706ab5edae5","_airbyte_emitted_at":1699949786022,"_airbyte_data":{}}
{"_airbyte_ab_id":"504e3a4e-1255-45ec-82b3-0e921ac42245","_airbyte_emitted_at":1699949786023,"_airbyte_data":{}}
...snip...
```

The issue resolves when I downgrade to version, 0.30.9.

**Comment by marcosmarxm** [comment]
What version of the Python CDK your connector is using?
Did you try to upgrade the connector or run the `read` command manually?

**Reply by enricojr01** [reply]
According to pip my connector is using 

```
#9 11.97 Downloading airbyte_cdk-0.53.8-py3-none-any.whl (613 kB)
#9 12.05    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 613.4/613.4 kB 8.4 MB/s eta 0:00:00
```

I have run the `read` command locally, as part of development, and it returns data just fine. We discovered this issue following a code change that only involved adding the `properties` key to each of the schema files (previously, you could omit it and it would build and work just fine)

**Reply by marcosmarxm** [reply]
Did you try to use another destination?

**Reply by enricojr01** [reply]
I haven't yet, no, I ended up downgrading to an older version because we needed the data

**Reply by marcosmarxm** [reply]
@cornjuliox Is it possible to share the custom connector code or any specific info about how to reproduce the issue?

**Reply by enricojr01** [reply]
@marcosmarxm Here's the connector [source code](https://github.com/Infinit-O/airbyte/tree/e742cad752601bd5920a1acc01cf11781d780e2e/airbyte-integrations/connectors/source-zoho-recruit)

As for how to reproduce, just build the connector and run through the Airbyte UI.

**Reply by shreyash-sacumen** [reply]
does anyone have ans for this ? I am facing the same issue

[a comment by an automated account is not shown]
